processor/filterprocessor/logs.go

```diff
@@ -19,9 +19,11 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/filter/filterlog"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/filter/filterottl"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/ottllog"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor/internal/common"
 )
 
 type filterLogProcessor struct {
+	consumers []common.LogsConsumer
```

Comment:
It would be great if we could add some tests for all signals processors using their context inferrer settings (like we do for the `transformprocessor` ([example](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/internal/logs/processor_test.go)), so we can ensure that filtering all possible inferred contexts works.